docs/src/guides/typo3/deploy/deploy.md

```diff
@@ -0,0 +1,102 @@
+---
+
+## Deployment
+
+If you have been customizing your existing project as you go, now is the time to ensure all code is committed to Git and to `git push` to Platform.sh on the `master` branch.  That will cause Platform.sh to build your code, producing a read-only file system image of your application, and then deploy it into a running cluster of containers.
+
+You can view the process from the Management Console; when it is done, click the URL shown in the Console to see your site.
+
+## Post-install (new site)
+
+
+You probably noticed the following lines of code in your `.platform.app.yaml` file during the deploy hook:
+
+```yaml
+    # The deploy hook runs after your application has been deployed and started.
+    # Code cannot be modified at this point but the database is available.
+    # The site is not accepting requests while this script runs so keep it
+    # fast.
+    deploy: |
+        # Exit hook immediately if a command exits with a non-zero status.
+        set -e
+
+        # Set TYPO3 site defaults on first deploy.
+        if [ ! -f var/platformsh.installed ]; then
+            # Copy the created LocalConfiguration into the writable location.
+            cp public/typo3conf/LocalConfiguration.FromSource.php var/LocalConfiguration.php
+
+            # On first install, create an inital admin user with a default password.
+            # *CHANGE THIS VALUE IMMEDIATELY AFTER INSTALLATION*
+            php vendor/bin/typo3cms install:setup \
+                --install-steps-config=src/SetupDatabase.yaml \
+                --site-setup-type=no \
+                --site-name="TYPO3 on Platform.sh" \
+                --admin-user-name=admin \
+                --admin-password=password \
+                --skip-extension-setup \
+                --no-interaction
+
+            # Sets up all extensions that are marked as active in the system.
+            php vendor/bin/typo3cms extension:setupactive || true
+
+            # Create file that indicates first deploy and installation has been completed.
+            touch var/platformsh.installed
+        fi;
+```
+
+The template is designed to run the TYPO3 installer only on the first deploy by creating a placeholder `platformsh.installed` on the `var` mount. You should not remove this file unless you want the next deployment to run through the installer again.
+
+Next, when the deploy hook ran through the installer, it set an intial username and password for the TYPO3 site, which you will want to update immediately. Visit `/typo3` on the generated url for the environment and login with those temporary `admin` credentials. Then click on the user icon in the top right of the page and go to User Settings > Password to update.
+
+Lastly you will need to enable the `pixelant/pxa-lpeh` extension you added with Composer. From the Backend, click on the Extensions icon in the sidebar. Search for the extension and then `Activate` it.
+
+## Data migration
```

Comment:
I'll get back to you with some housekeeping functions here, such as running database:updateschema to make sure the database schema is OK.